app/controllers/categories_controller.rb

```diff
@@ -2,4 +2,8 @@ class CategoriesController < ApplicationController
 	def show
 		@category = Category.includes(:creatures).find_by(name: params[:category])
 	end
-end
\ No newline at end of file
+
+	def index
+		@categories = Category.all
```

Comment:
Here is where we'd probably use includes, something like `Category.includes(:creatures).all`